docs/config.md

```diff
@@ -318,6 +319,100 @@ location /blog/ {
 
 Default: `[]`
 
+- `NoDNSLink`  
+  A boolean to configure whether DNSLink lookup for value in `Host`
+  HTTP header should be performed.  If DNSLink is present, content path stored
+  in the DNS TXT record is mounted at `/` and respective payload is returned to
+  the client.
+
+Default: `false`
+
+### Gateway.PublicGateways
+
+`PublicGateways` is a dictionary for defining gateway behavior on specific fully qualified domain names (FQDN).  
+Options for changing behavior per hostname:
+
+- `Paths`  
+  Array of paths that should be mounted at the root of domain name.  
+  Example: `["/ipfs", "/ipns", "/api"]`  
+  - **Note:** when both `Paths` and DNSLink are mounted, `Paths` takes priority
```

Comment:
DNSLink mounted?

Reply:
Yeah not sure what is the best wording here, would appreciate suggestions:

```json
     "PublicGateways": {
        "example.com": {
          "Paths": ["/ipfs"],
          "NoDNSLink": false,
```

Above resolves DNSLink for `example.com` and mounts it at `/`, then mounts `/ipfs/` on top of it, so content-addressed gateway is used instead of a DNSLink subpath. 

Would below be better?

```suggestion
  - **Note:** If DNSLink is enabled `Paths` are mounted on top of `/` and take priority over DNSLink.
```

Reply:
That sounds reasonable. It's just that "DNSLink is mounted" is a bit confusing.